src/main/java/org/eln2/mc/extensions/Vector4fExtensions.kt

```diff
@@ -0,0 +1,10 @@
+package org.eln2.mc.extensions
+
+import com.mojang.math.Vector3f
+import com.mojang.math.Vector4f
+
+object Vector4fExtensions {
```

Comment:
Ah, function extensions! I don't believe you need to scope these to an object; the "file scope" is implicitly a class :)